.github/workflows/release.yaml

```
@@ -60,25 +60,31 @@ jobs:
 
       - name: Build binary and make tarball
         id: build_bin
+        env:
+          RELEASE_VERSION: ${{ steps.get_tag.outputs.release_version }}
         run: |
           make bin
-          TARBALL_NAME="chart-verifier-${{ steps.get_tag.outputs.release_version }}.tgz"
-          tar -zcvf $TARBALL_NAME -C out/ chart-verifier
-          export TARBALL_PATH=$(realpath $TARBALL_NAME)
+          TARBALL_NAME="chart-verifier-${RELEASE_VERSION}.tgz"
+          tar -zcvf "$TARBALL_NAME" -C out/ chart-verifier
+          export TARBALL_PATH=$(realpath "$TARBALL_NAME")
           echo "tarball_path=$TARBALL_PATH" | tee -a $GITHUB_OUTPUT
 
       - name: Check that the tag matches the current version
         id: check_tag_and_version
+        env:
+          RELEASE_VERSION: ${{ steps.get_tag.outputs.release_version }}
         run: |
-          release_version=${{ steps.get_tag.outputs.release_version }}
           bin_version=$(out/chart-verifier version --as-data | jq -r .version)
-          if [[ "$release_version" != "$bin_version" ]]; then
-            echo "Binary version ($bin_version) doesn't match tag ($release_version)" && exit 1
+          if [[ "${RELEASE_VERSION}" != "$bin_version" ]]; then
+            echo "Binary version ($bin_version) doesn't match tag (${RELEASE_VERSION})" && exit 1
           fi
 
       - name: Generate SBOM filename
         id: generate_sbom_filename
-        run: echo sbom_filename="${{ github.event.repository.name }}-${{ steps.get_tag.outputs.release_version }}-sbom.spdx.json" | tee -a $GITHUB_OUTPUT
+        env:
+          REPOSITORY_NAME: ${{ github.event.repository.name }}
+          RELEASE_VERSION: ${{ steps.get_tag.outputs.release_version }}
+        run: echo sbom_filename="${REPOSITORY_NAME}-${REPOSITORY_VERSION}-sbom.spdx.json" | tee -a $GITHUB_OUTPUT
```

Variable name mismatch, I think?

```suggestion
        run: echo sbom_filename="${REPOSITORY_NAME}-${RELEASE_VERSION}-sbom.spdx.json" | tee -a $GITHUB_OUTPUT
```